phpunit-10.xml.dist

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Since this file is not being used by default and the tests seem to pass (just config file deprecations), should we remove?

Alternatively, I think if we rename this to phpunit.dist.xml it will be picked up by phpunit 10-only (need to confirm).

Reply:
Actually, under phpunit 9 it triggers a warning but does execute tests without problems : 

```
PHPUnit 9.5.0 by Sebastian Bergmann and contributors.

  Warning - The configuration file did not pass validation!
  The following problems have been detected:

  Line 25:
  - Element 'source': This element is not expected.

  Test results may not be as expected.


...............................................................  63 / 284 ( 22%)
I.............................................................. 126 / 284 ( 44%)
............................................................... 189 / 284 ( 66%)

...